plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,526 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2021, Tyler Gates <tgates81@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: spectrum_model_attrs
+short_description: Enforce a model's attributes in CA Spectrum.
+description:
+    - This module can be used to enforce a model's attributes in CA Spectrum.
+version_added: 2.1.0
+author:
+    - Tyler Gates (@tgates81)
+notes:
+    - Tested on CA Spectrum version 10.4.2.0.189
+    - Model creation and deletion are not possible with this module. For that use M(spectrum_device) instead.
+requirements:
+    - 'python >= 2.7'
+options:
+   url:
+     description:
+     - URL of OneClick server.
+     type: str
+     required: True
```

Comment:
```suggestion
     required: true
```
Please use `true`/`false` in YAML.